pkg/object/search.go

```diff
@@ -173,6 +173,13 @@ func (f *SearchFilters) addReservedFilter(op SearchMatchType, keyTyp filterKeyTy
 	f.addFilter(op, keyTyp, "", val)
 }
 
+// addFlagFilters adds filters that works like flags: they don't need to have
+// specific match type or value. They processed by NeoFS nodes by the fact
+// of presence in search query. E.g.: PHY, ROOT.
+func (f *SearchFilters) addFlagFilter(keyTyp filterKeyType) {
+	f.addFilter(0, keyTyp, "", staticStringer(""))
```

Comment:
I suggest to replace `0` with `MatchUnknown`

Reply:
Agree.